src/dbcp/extract/manual_ordinances.py

```diff
@@ -1,30 +1,44 @@
 """Load manually maintained ordinances from BigQuery."""
-import google.auth
+
 import pandas as pd
-from pandas_gbq import read_gbq
+
+from dbcp.extract.helpers import cache_gcs_archive_file_locally, extract_airtable_data
 
 
-def extract() -> dict[str, pd.DataFrame]:
+def extract(counties_uri: str) -> dict[str, pd.DataFrame]:
     """Extract manually maintained ordinances from BigQuery.
 
     Returns:
         dfs: dictionary of dataframe name to raw dataframe.
     """
     dfs = {}
-    credentials, project_id = google.auth.default()
-    dfs["manual_ordinances"] = read_gbq(
-        """SELECT
-    county_id_fips,
-    CASE
-        WHEN ordinance_status IS NULL THEN NULL
-        WHEN ordinance_status IN ('Ordinance/moratorium change in process', 'Prohibitive ordinance/moratorium') THEN TRUE
-        ELSE FALSE
-    END as ordinance_via_self_maintained
-    FROM `local-jobs-econ-dev-fund.airtable_data.county_permitting_info`
-    """,
-        project_id=project_id,
-        credentials=credentials,
+    counties_uri, counties_generation_num = str(counties_uri).split("#")
```

Comment:
nit: It seems like it might be nicer to have the URI and generation number be separate params, then we don't have to have this `#` split situation.